nixos/modules/module-list.nix

```diff
@@ -241,11 +241,12 @@
   ./services/mail/opendkim.nix
   ./services/mail/opensmtpd.nix
   ./services/mail/postfix.nix
-  ./services/mail/postsrsd.nix
   ./services/mail/postgrey.nix
-  ./services/mail/spamassassin.nix
-  ./services/mail/rspamd.nix
+  ./services/mail/postsrsd.nix
   ./services/mail/rmilter.nix
+  ./services/mail/rspamd.nix
+  ./services/mail/spamassassin.nix
+  ./services/mail/nullmailer.nix
```

Comment:
Yeah. I sorted the surrounding ... but you are right on second thought, sorting the list should be done in a seperated patch.